cli/create.go

```diff
@@ -41,11 +42,23 @@ func (cc *CreateCommand) addFlags() {
 	flagSet.StringVar(&cc.name, "name", "", "Specify name of container")
 	flagSet.BoolVarP(&cc.tty, "tty", "t", false, "Allocate a tty device")
 	flagSet.StringSliceVarP(&cc.volume, "volume", "v", nil, "Bind mount volumes to container")
+	flagSet.StringVar(&cc.runtime, "runtime", "", "Specify oci runtime")
+	flagSet.StringVar(&cc.platform, "platform", runtime.GOOS, "Specify the platform for running container")
```

Comment:
Actually I am not sure whether we should put `platform` into a flag in command line side. since it should always be `runtime.GOOS`, We could use this just in ctrd. This is sufficient. WDYT? @Ace-Tang